### PR TITLE
patching corpus query

### PR DESCRIPTION
**Pull request by benlower**
Changing the body to use 'max_chunks'.

### CODE REVIEW

File: packages/ai-jsx/src/batteries/docs.tsx [hunk 1]
```diff
@@ -645,7 +645,7 @@ export class FixieCorpus<ChunkMetadata extends Jsonifiable = Jsonifiable> implem
       body: JSON.stringify({
         corpus_id: this.corpusId,
         query,
-        page_size: params?.limit ?? 10,
+        max_chunks: params?.limit ?? 10,
```
Comment (mdepinet) [inline packages/ai-jsx/src/batteries/docs.tsx:648]: We should really be using camelCase I think, but if this works then I suppose I don't really care either way. :)

**Review by mdepinet — APPROVED** [review]
Thanks!

File: packages/ai-jsx/src/batteries/docs.tsx (hunk 1 above)
Comment (benlower) [reply packages/ai-jsx/src/batteries/docs.tsx:648]: i don't know what the right way is. i just copied the format that was used here https://github.com/fixie-ai/fixie/blob/4d7f23adaa585a01e408f8f68a05574800a9db08/src/proto_api/fixie_api/corpus/v1/service.proto#L152C22-L152C22

Comment (mdepinet) [reply packages/ai-jsx/src/batteries/docs.tsx:648]: Fair enough. The OpenAPI spec we generate uses camelCase for field names, but our server will parse either one.

File: packages/ai-jsx/src/batteries/docs.tsx [hunk 2]
```diff
@@ -645,7 +645,7 @@ export class FixieCorpus<ChunkMetadata extends Jsonifiable = Jsonifiable> implem
...
       }),
     });
     if (response.status !== 200) {
```
Comment (mdepinet) [inline packages/ai-jsx/src/batteries/docs.tsx:651]: Could you update line 661 to use result.chunkContent while you're here please?

**Review by mdepinet — APPROVED** [review]

**Review by mdwelsh — APPROVED** [review]

File: packages/ai-jsx/src/batteries/docs.tsx (hunk 2 above)
Comment (benlower) [reply packages/ai-jsx/src/batteries/docs.tsx:651]: done

**Pull request merged**